nilearn/datasets/func.py

```diff
@@ -206,6 +143,13 @@ def fetch_haxby(data_dir=None, n_subjects=1, fetch_stimuli=False,
             **kwargs)
 
 
+@deprecated("fetch_haxby_simple will be removed in future releases. "
+            "Calling 'fetch_haxby' instead.")
```

Comment:
Makes sense. I restored the old function and put a note in the deprecated flag saying the user should use `fetch_haxby` instead.